packages/venia-ui/lib/components/MiniCart/body.js

```diff
@@ -34,13 +34,17 @@ const Body = props => {
     const classes = mergeClasses(defaultClasses, props.classes);
 
     // Callbacks.
-    const handleEditItem = useCallback(
+    const handleBeginEditItem = useCallback(
         item => {
-            beginEditItem(item);
+            beginEditItem();
+            setEditItem(item);
```

Comment:
[beginEditItem](https://github.com/magento/pwa-studio/blob/fa0a4c9b45757c272f6b745c546e8b61b1da40db/packages/venia-ui/lib/reducers/cart.js#L122) is entirely view state, we may be able to get rid of it here entirely (looks like it also sets `isEditingItem`).

Reply:
The only thing it does right now is `setIsEditingItem` to `false` in the root miniCart component. Instead of threading setters/state down I just modified the callback.